### PR TITLE
Port arithmatic test

### PR DESCRIPTION
**Pull request by trijobe**
This branch replaces ArithmaticTest.java with ArithmaticTest.kt, a kotlinized version.

### CODE REVIEW

File: src/test/kotlin/com/avail/test/ArithmeticTest.kt [hunk 1]
```diff
@@ -0,0 +1,528 @@
...
+		private fun checkBitShift(
+			base: BigInteger,
+			leftShift: Int)
+		{
+			val availInt = fromBigInteger(base)
+			val availShift: A_Number = fromInt(leftShift)
+			val shiftedAvailInt = availInt.bitShift(availShift, true)
+			val availInt2 = fromBigInteger(base)
+			val shiftedAvailInt2 = availInt2.bitShift(availShift,
+													  false)
```
Comment (toddATavail) [inline src/test/kotlin/com/avail/test/ArithmeticTest.kt:488]: Move to the previous line.

File: .idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -1,138 +1,83 @@
 <component name="ProjectCodeStyleConfiguration">
```
[the remark this answers is not in the document]
Comment (trijobe) [reply .idea/codeStyles/Project.xml]: I think it's excluded now, but not 100% sure if I did it right. Is the latest commit correct?

File: src/test/kotlin/com/avail/test/ArithmeticTest.kt [hunk 3]
```diff
@@ -0,0 +1,528 @@
...
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.avail.test
+
+import com.avail.descriptor.numbers.A_Number
+import com.avail.descriptor.numbers.DoubleDescriptor.Companion
+.doubleTruncatedToExtendedInteger
```
[the remark this answers is not in the document]
Comment (trijobe) [reply src/test/kotlin/com/avail/test/ArithmeticTest.kt]: Done

File: src/test/kotlin/com/avail/test/ArithmeticTest.kt (hunk 1 above)
Comment (trijobe) [reply src/test/kotlin/com/avail/test/ArithmeticTest.kt]: Done.

File: .idea/codeStyles/Project.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (toddATavail) [reply .idea/codeStyles/Project.xml]: It looks like it is trying to delete the file from the project altogether; it did more than just remove the file from the commit. So you might want to research removing a file from a commit without deleting it from the project; I'm afraid I don't know the answer off the top of my head.

**Pull request merged**